lib/iris/analysis/__init__.py

```diff
@@ -1597,7 +1601,13 @@ def regridder(self, src_grid, target_grid):
                 `callable(cube)`
 
             where `cube` is a cube with the same grid as `src_grid`
-            which is to be regridded to the `target_grid`.
+            that is to be regridded to the `target_grid`.
+            Thus for the callable returned by
+            `Linear().regridder(src_grid, target_grid)`, cube must
+            be on the same grid as src_grid.
+
+            This callable would typically be used by
+            :class:`iris.cube.Cube.regrid`.
```

Comment:
:+1: Thanks @dkillick 

> regridder = Linear().regridder(src_grid, target_grid) for cube in my_long_list_of_cubes_on_the_src_grid: cube_on_target_grid = regridder(cube) # Do stuff with cube_on_target_grid ...

I'd second highlighting the above use pattern in the doc-string as it may not be immediately obvious to the user.